### DISCUSSION TITLE
[Feature Request]  Option to hide syncing messages

### DISCUSSION THREAD

**Discussion by fra-gadio** [remotely-save/remotely-save#536] [Ideas]
I have version 0.3.40.  Often as I'm typing I see the syncing message popping up and I find it quite distracting.   If there was an option to hide the syncing messages (and maybe just show a message if there's a syncing error)  it would be greatly appreciated.

**Comment by nallebrean** [comment]
Well it's actually a problem that shows up for me. The messages are shown when there is another syncing session still in action. I have flipped around with the settings for syncing (interval, at start up and after saving detected). Sync when the app starts and every minute works best for me but I still get som error messages. Because there is no diff sync I want high frequent updates because I switch dev often. I wish I could stop them from showing up unless there is another problem. The messages occur on all my devices. 2 different androids and windows. 

Version 0.3.40.

**Comment by nallebrean** [comment]
I the latest 0.4.16 the sync messages when another sync session are active are gone. We can close this because it's the same problem.
